src/schemas/role.graphql

```diff
@@ -0,0 +1,100 @@
+"The fields by which you are allowed to sort any query that returns an `RoleConnection`"
+enum RoleSortByField {
+  "Role ID"
+  _id
+
+  "Role name"
+  name
+}
+
+"Represents a named role"
+type Role implements Node {
```

Comment:
Would it be feasible to rename `Role` to `SimpleRole`? While it's true you realistically wouldn't want this plugin to be used along with its successor `plugin-authorization`, this type collision is actually the only thing preventing them from both being loaded.

We have a [PR prepared](https://github.com/reactioncommerce/reaction-authorization/pull/132) in case you don't want to change this, in which we change authorization's `Role` to `AuthorizationRole`, but I think it makes more sense to change this simple/legacy plugin instead.

@kieckhafer do you have thoughts on this as well?

Reply:
@rosshadden It's not that I don't want to change it, but I can't see how we could do so without needing a major version bump due to breaking backward compatibility. And it doesn't seem worthy of breaking since it's unlikely to conflict.